packages/manager/cypress/e2e/core/linodes/clone-linode.spec.ts

```diff
@@ -113,6 +131,167 @@ describe('clone linode', () => {
+    mockGetLinodeConfigs(clonedLinode.id, [mockConfig]).as('getLinodeConfigs');
+    cy.visitWithLogin('/linodes/create');
+
+    // Fill out necessary Linode create fields.
+    linodeCreatePage.selectRegionById(mockLinodeRegion.id);
+    linodeCreatePage.selectImage('Debian 11');
+    linodeCreatePage.setLabel(mockLinode.label);
+    linodeCreatePage.selectPlan('Shared CPU', 'Nanode 1 GB');
+    linodeCreatePage.setRootPassword(randomString(32));
+
+    // Open VLAN accordion and select existing VLAN.
+    ui.accordionHeading.findByTitle('VLAN').click();
+    ui.accordion
+      .findByTitle('VLAN')
+      .scrollIntoView()
+      .should('be.visible')
+      .within(() => {
+        cy.findByLabelText('VLAN').should('be.enabled').type(mockVlan.label);
+
+        ui.autocompletePopper
+          .findByTitle(mockVlan.label)
+          .should('be.visible')
+          .click();
+
+        cy.findByLabelText(/IPAM Address/)
+          .should('be.enabled')
+          .type(mockVlan.cidr_block);
+      });
+
+    // Confirm that VLAN attachment is listed in summary, then create Linode.
+    cy.get('[data-qa-linode-create-summary]')
+      .scrollIntoView()
+      .within(() => {
+        cy.findByText('VLAN Attached').should('be.visible');
+      });
+
+    ui.button
+      .findByTitle('Create Linode')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    // Confirm outgoing API request payload has expected data.
+    cy.wait('@createLinode').then((xhr) => {
+      const requestPayload = xhr.request.body;
+      const expectedPublicInterface = requestPayload['interfaces'][0];
+      const expectedVlanInterface = requestPayload['interfaces'][1];
+
+      // Confirm that first interface is for public internet.
+      expect(expectedPublicInterface['purpose']).to.equal('public');
+
+      // Confirm that second interface is our chosen VLAN.
+      expect(expectedVlanInterface['purpose']).to.equal('vlan');
+      expect(expectedVlanInterface['label']).to.equal(mockVlan.label);
+      expect(expectedVlanInterface['ipam_address']).to.equal(
+        mockVlan.cidr_block
+      );
+    });
+
+    cy.url().should('endWith', `/linodes/${mockLinode.id}`);
+    // Confirm toast notification should appear on Linode create.
+    ui.toast.assertMessage(`Your Linode ${mockLinode.label} is being created.`);
+
+    mockCloneLinode(mockLinode.id, clonedLinode).as('cloneLinode');
+    cy.visitWithLogin(`/linodes/${mockLinode.id}`);
+
+    // Wait for Linode to boot, then initiate clone flow.
+    cy.findByText('OFFLINE').should('be.visible');
+
+    ui.actionMenu
+      .findByTitle(`Action menu for Linode ${mockLinode.label}`)
+      .should('be.visible')
+      .click();
+
+    ui.actionMenuItem.findByTitle('Clone').should('be.visible').click();
+    const url = getLinodeCloneUrl(mockLinode);
+    console.log(`linode clone url: ${url}`);
+    cy.url().should('endWith', getLinodeCloneUrl(mockLinode));
```

Comment:
do we still need this console.log? if not we can consolidate a little
```suggestion
    cy.url().should('endWith', getLinodeCloneUrl(mockLinode));
```